appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -2417,6 +2441,17 @@ public void setTutorialVisible(boolean visible) {
     }
   }
 
+  public void setHelpVisible(boolean visible){
```

Comment:
Make sure to add Javadocs for the public methods. Note that there should also be a space between the `)` and the `{`